### ISSUE TITLE
AuthorName in Inbox not working

### ISSUE THREAD

**Issue by IzzySoft** [soscripted/sox#415]
As you can see, I have the option enabled to add AuthorName to messages in Inbox. This doesn't seem to work anymore (I remember it did once in the past):

![image](https://user-images.githubusercontent.com/6781438/61087848-17257c80-a437-11e9-9c00-20bfe6ab32ab.png)

Doesn't matter whether I tick the "show name before content" or not. Relevant lines from debug:

~~~
SOX: running addAuthorNameToInboxNotifications
SOX: running alignBadgesByClass
~~~

(no errors, warnings, whatever between those two lines) Just updated to the latest version, no change.

Besides: The cog/wheel doesn't show up in the MultiCollider™ anymore either: I have to guess its location to enter settings. It seems it needs some additional external JS as it's showing up when I unblock cdn.jsdelivr.net – that intended? What is there that couldn't be installed locally? Ya know, me tin foil :smile:

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-alignBadgesByClass","Appearance-colorAnswerer","Appearance-dragBounty","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-scrollToTop","Appearance-spoilerTip","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Appearance-showTagWikiLinkOnTagPopup","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-hiddenCommentsIndicator","Comments-onlyShowCommentActionsOnHover","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideCommunityBulletin","Sidebar-hideJustHotMetaPosts","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-linkedToFrom","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Voting-betterCSS","Voting-stickyVoteButtons","Voting-disableVoteButtons","Extras-linkedPostsInline","Extras-quickAuthorInfo","Extras-sortByBountyAmount","Extras-warnNotLoggedIn","Extras-hideCertainQuestions","Extras-showMetaReviewCount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab"]

**Comment by shu8** [comment]
@IzzySoft thanks for reporting this, I hadn't noticed!

Fixed in [dev 2.5.17](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)

**Comment by shu8** [comment]
SOX icons issue fixed in [dev 2.5.18](https://github.com/soscripted/sox/raw/dev/sox.user.js) too :)

**Comment by IzzySoft** [comment]
Fast as usual – Thanks! Confirming both solved in dev 2.5.18. :+1:
